src/index.html

```diff
@@ -20,8 +21,404 @@
       href="styles/index.scss"
     />
   </head>
+  <!-- #endregion -->
 
   <body>
-    <h1>Catalog</h1>
+    <!-- #region header -->
+    <header class="header">
+      <a
+        href="#home"
+        class="logo-link"
+      >
+        <img
+          src="images/logo.png"
+          alt="Moyo logo"
+          class="logo"
+        />
+      </a>
+
+      <nav class="nav-links">
+        <ul class="nav-items">
```

Comment:
```suggestion
        <ul class="nav__list">
```
use correct BEM naming (example means that nav is a separate BEM block so its styles should be in a separate file)

Reply:
use correct BEM naming everywhere